packages/p2p/src/components/my-ads/form-ads.jsx

```diff
@@ -255,6 +259,15 @@ const FormAds = observer(() => {
                     );
                 }}
             </Formik>
+            <Dialog
+                title={localize('Something’s not right')}
+                is_visible={is_warning_visible}
+                confirm_button_text={localize('OK')}
+                onConfirm={() => setDialogOpen(false)}
+                is_mobile_full_width={false}
+            >
+                <div>{my_ads_store.api_error_message}</div>
```

Comment:
Should this be wrapped in a `Text` component (try to test on both dark and light themes)?